### PR TITLE
Create RishitaxGhosh.md

### PR DESCRIPTION
**Pull request by RishitaxGhosh**
## Describe your project in 1-3 sentences
We are building a braille embosser & Walk bot for the visually impaired!

## Hack Club Slack handle
Rishita Ghosh 
Harpuneet 
Kaustav Ghosh
Akshita Pant 

## Checklist before submitting a proposal:

- [X] I am a current high school student (or younger).
- [X] I have submitted an RSVP at <https://hackclub.com/winter>
- [X] I am a member of the Hack Club Slack (Join at <https://hackclub.com/slack>)
- [X] I have self-reviewed my project proposal.

### CODE REVIEW

**Comment by bellesea** [comment]
heyy thanks so much for making a PR! **we're back on tuesday and will start reviewing PRs in the order that they came in, thanks for your patience :)**

to make the process smoother, please ensure you double checked that your file is properly formatted, you have all the parts you need, and that your plan is detailed enough for us to imagine how you're using the components in your budget to build your project and visualise you work on it :)

and fyi, we're looking for projects where/that:
- you can spend at least 10 days building (from Feb 14th to 23rd)
- you are able to modify or make your own in some ways
- you can learn something new from
- involves working with electronics or software on a lower level than a pre-assembled kit of parts & ready software

and, we normally don’t approve projects where you buy consumer components w/o modification like keyboards (buying PCB, keycaps, switches, and assembling it), PCs (buying motherboard, cpu, memory and plugging it together), and kits (eg. closed-sourced consumer drone kits). 

(note: can all of you please respond to this PR confirming that you individually meet the checklist for proposal? thank you!)

our Jan 15 deadline is for PRs to come in and not for final approval so we'll work with you to either accept this project OR give you a chance to submit another PR. 

if you have any questions, please do not hesitate to ask her or DM me (i'm belle on slack), thank you!

**Review by cfanoulis — CHANGES_REQUESTED** [review]
Hey!! Thanks for your proposal - before we go ahead, can we please go thru the following?
 - Can you please include a more in-depth plan of your walk-bot? I see a lot of items mentioned on the instructable you listed that are missing from your budget - so I'm assuming it's not the same
 - Can you please make the braille embosser include some form of electronics? Maybe you could try mechanzing it using servos and timing belts, like those old dot-matrix printers!
 - I see you included a soil moisture sensor - what does this serve for?

Also, please have your teammates complete the proposal checklist, including their Slack usernames.

**Comment by RishitaxGhosh** [comment]
> Hey!! Thanks for your proposal - before we go ahead, can we please go thru the following?
> 
> * Can you please include a more in-depth plan of your walk-bot? I see a lot of items mentioned on the instructable you listed that are missing from your budget - so I'm assuming it's not the same
> * Can you please make the braille embosser include some form of electronics? Maybe you could try mechanzing it using servos and timing belts, like those old dot-matrix printers!
> * I see you included a soil moisture sensor - what does this serve for?
> 
> Also, please have your teammates complete the proposal checklist, including their Slack usernames.

**Comment by RishitaxGhosh** [comment]
Hi, apologies for the late reply. I have asked my members to verify, they will soon do it. 

As for the walk bot, I will be using a couple of instruction manuals: 
1. https://github.com/harshgondaliya/Multi-feature-Smart-Stick-for-Blind 
2. https://hackaday.io/project/189009-walk-bot
The soil moisture sensor will be used as a water sensor, where if the stick touches any liquid, the buzzer will ring to alert the visually impaired person. 

About the braille embosser, it is certainly a great suggestion! I would love to re-engineer the embosser dial with a timing belt, servo motor and buttons to make it an automatic control system.

**Comment by Harpuneet2005** [comment]
Hello, 
My name is Harpuneet Singh and I am confirming my participation. I have completed the checklist.
My Slack ID is U04AN502W58 and my Display Name is Harpuneet Singh.
My email is harpuneet20@gmail.com

**Comment by cfanoulis** [comment]
hey - I'll need your team to complete the same checklist as you did in your PR description

also, please don't include your email hahahahah let's keep those private

**Comment by Kaustav-Ghosh** [comment]
- [X]  I am a current high school student (or younger).
- [X]  I have submitted an RSVP at https://hackclub.com/winter
- [X]  I am a member of the Hack Club Slack (Join at https://hackclub.com/slack)
- [X]  I have self-reviewed my project proposal.
Hello, my name is Kaustav, Rishita's brother. I am currently in middle school and super excited to build, break and create this project with all my team mates! My slack display name is: Kaustav Ghosh

**Comment by Harpuneet2005** [comment]
ahh ok, sorry my bad,

- [X]  I am a current high school student (or younger).
- [X]  I have submitted an RSVP at https://hackclub.com/winter
- [X]  I am a member of the Hack Club Slack (Join at https://hackclub.com/slack)
- [X]  I have self-reviewed my project proposal.

Hello, My name is Harpuneet Singh and I am confirming my participation. I have completed the checklist.
My Slack ID is U04AN502W58 and my Display Name is Harpuneet Singh.

**Comment by Akshita-Pant-design** [comment]
- [X]  I am a current high school student (or younger).
- [X]  I have submitted an RSVP at https://hackclub.com/winter
- [X]  I am a member of the Hack Club Slack (Join at https://hackclub.com/slack)
- [X]  I have self-reviewed my project proposal.

Hi, I'm Akshita, I am also very excited to work on this project. My slack ID: Akshita Pant

**Comment by cfanoulis** [comment]
heyy thanks for that! can you please update your proposal on how you plan to motorize the embosser? thanks

**Comment by cfanoulis** [comment]
Hi! As we need to wrap up on proposals this week, please reply to my request for changes within the next 48 hours - else we will be closing this PR.

File: RishitaxGhosh.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+| PLA Filament - Yellow | https://amzn.eu/d/2qIgAZW | $15 | 
+| Multimeter | https://amzn.eu/d/gX4CFgO | $40 |
+| Soldering Iron | https://amzn.eu/d/awg4gd1 | $ 23 | 
+| Arduino UNO R3  | https://amzn.eu/d/grABcyf             | $15 |
+| Buzzers         | https://amzn.eu/d/377hADx             | $5 |
+| Breadboard + resistor + battery | https://amzn.eu/d/377hADx | $15 | 
+| Jumper Wires | https://amzn.eu/d/b1e26H2 | $5 | 
+| Soil Humidity Sensor | https://amzn.eu/d/h1Ukyn6 | $5 | 
+| IR Remote control | https://amzn.eu/d/3MbbMJr | $5 | 
+| PVC pipe & Aluminium rod | (will check from shop) | $30 (approx) |
```
Comment (Harpuneet2005) [inline RishitaxGhosh.md:60]: +| Screwdriver | https://www.amazon.in/dp/B07V3TDP49   | $9 |

File: RishitaxGhosh.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+| Total           |                                       | $588 |
```
Comment (Harpuneet2005) [inline RishitaxGhosh.md:61]: | Total           |                                       | $597 |

**Comment by Harpuneet2005** [comment]
Hi, 
A slight change of plans, 
About the motorization of the embosser, we plan to use a servo motor and a rack and pinion gear to lower the embosser and press it into the paper. It should improve consistency and prevent accidental tearing of the paper.

**Comment by RishitaxGhosh** [comment]
Updated the file with changes!

**Review by cfanoulis — APPROVED** [review]
perfect, tysm - let's get you your grant. someone from @hackclub/bank-ops will be in touch to get you setup on bank

**Review by hackliv — APPROVED** [review]
Wow this is so cool!! @RishitaxGhosh @Kaustav-Ghosh @Akshita-Pant-design @Harpuneet2005 I'm dming you all on Slack now to see if we can set up some time to meet and chat about next steps for the grant!

**Pull request merged**